packages/mongo-nav/src/MongoNav.spec.tsx

```diff
@@ -123,6 +124,52 @@ describe('packages/mongo-nav', () => {
     });
   });
 
+  describe('reloadData API is exposed to consumer through ref passed to MongoNav', () => {
+    let ref: React.RefObject<any>;
+
+    const MongoNavWrapper = () => {
+      ref = React.useRef();
+
+      return <MongoNav ref={ref} />;
+    };
+
+    const renderWrapperComponent = async () => {
+      await act(async () => {
+        render(<MongoNavWrapper />);
+      });
+    };
+
+    const responseObject = {
+      ok: true,
+      json: () => Promise.resolve(dataFixtures),
+    };
+
+    beforeEach(async () => {
+      fetchMock.mockResolvedValue(responseObject);
+
+      await renderWrapperComponent();
+
+      act(() => {
+        responseObject.json();
+      });
```

Comment:
can we reset the mock here?

Reply:
yep!

Reply:
if we reset it we can remove the comment below
>// twice, once for main data and once for alerts polling

Reply:
Updated! Let me know if this is what you had in mind